### PR TITLE
[CSS] add scroll to live visits table

### LINKED ISSUE #19254: Fix adjacent elements to dashboard-dropdown-menu-modal overlapping on narrower screens
## A user reported:

_We found an issue on the Matomo Page Stats Dashboard. When you click on the Dashboard button, the submenu is under lapping of the date bar and channel types bar._ 

bug-modal-is overwritten

## I confirmed 

For a narrow screen, some of the divs escape from Visits in Real-time widget under these conditions:
* screen width 800 to 630 px roughly
* Firefox or Chromium browser, in Linux Ubuntu 
* The widget was "Visits in Real Time"
* my Matomo version was  4.10.0

tatll-overflow div

### PR DESCRIPTION
**Pull request by peterhashair**
### Description:

Fixes: #19254 
add scroll to live table

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by bx80 — CHANGES_REQUESTED** [review]
This solves the issue for my tests :+1: 

Since this issue is a regression it would be good to add a simple UI test screenshot of the dashboard with the window set to a size that causes the visits table header to overflow.

File: plugins/Live/stylesheets/live.less [hunk 1]
```diff
@@ -69,35 +70,39 @@
...
 
 .theWidgetContent .card .card-content #visitsTotal {
-  margin-left: -20px;
-  width: ~"calc(100% + 40px)";
+    margin-left: -20px;
+    width: ~"calc(100% + 40px)";
+}
+
+#visitsTotal {
+    overflow-x: scroll;
```
Comment (sgiehl) [inline plugins/Live/stylesheets/live.less]: This will actually always add a scrollbar to the widget. Would be good to only have it, if it is needed.

**Comment by bx80** [comment]
I've bumped the UI tests to run again, just want to be sure that the Movers and Shakers widget overflow is a random test failure and not something else.

**Comment by bx80** [comment]
The small screen dashboard UI test movers and shakers overflow is still failing each time on Travis but it passes when run locally, I've increase the delay from 250ms to 500ms to see if that fixes it.

**Review by bx80 — APPROVED** [review]

**Pull request merged**